deepcell/utils/plot_utils.py

```diff
@@ -59,16 +59,19 @@ def get_js_video(images, batch=0, channel=0, cmap='jet', vmin=0, vmax=30):
         'cmap': cmap,
     }
 
+    if vmax == 0:
+        vmax = images.max()
+
     # TODO: do these not work for other cmaps?
-    if cmap == 'cubehelix':
+    if cmap == 'cubehelix' or cmap == 'jet':
         plot_kwargs['vmin'] = vmin
         plot_kwargs['vmax'] = vmax
 
     for i in range(images.shape[1]):
         im = plt.imshow(images[batch, i, :, :, channel], **plot_kwargs)
         ims.append([im])
 
-    ani = animation.ArtistAnimation(fig, ims, interval=150, repeat_delay=1000)
+    ani = animation.ArtistAnimation(fig, ims, interval=interval, repeat_delay=1000)
```

Comment:
Should we add `repeat_delay` while we're at it?

Reply:
Might as well, done